src/components/Keyboard.tsx

```diff
@@ -65,24 +67,30 @@ export function Keyboard() {
     }
   }
 
+  function handleMouse() {
+    setKeysClicked(getState().keysClicked)
+  }
+
   return (
     <div
       onKeyDown={handleKeyDown}
       onKeyUp={handleKeyUp}
+      onMouseDown={handleMouse}
+      onMouseUp={handleMouse}
     >
       <PowerButton onClick={handleClickPowerButton} power={power} />
       <p>Synth is {synthList.length > 0 ? 'loaded' : 'not loaded: click the red power button'}</p>
       <div
         className={styles.keyboard}
       >
-        <SynthKey synth={synthList[0]} keyboardKey="a" pitch={`C${octave}`} keyDown={keysPressed.includes('a')} />
```

Comment:
Can we add {`C${octave}`} syntax back in for `pitch`? This is needed to change the text on the keys to reflect the octave.